src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -133,19 +133,23 @@ public void querySnapshotTest() throws Exception {
         SearchHits mockHits = mock(SearchHits.class);
         SearchHit mockHit = mock(SearchHit.class);
         when(mockHits.iterator()).thenReturn(Arrays.stream(new SearchHit[]{mockHit}).iterator());
-        when(mockHit.getSourceAsMap()).thenReturn(Map.of("testKey", "testValue"));
+        when(mockHit.getSourceAsMap()).thenReturn(Map.of(
+            "uuid", "41b203f2-c7a5-4232-bcb3-f3442ba9baaf",
```

Comment:
It's a minor thing but I would use named constants for the things that supposed to be the same, such as this uuid, to make it clearer to the reader.

Reply:
Can you clarify how this would work?

Reply:
Something like `testId` in `querySnapshotTest()`. Using an name for the UUID means that a reader doesn't have to examine the string `41b203f2-c7a5-4232-bcb3-f3442ba9baaf` in both places and see that it's the same thing.

Another way to do this would be to build this map from the map in the expected result by encoding its keys, that way the keys and values would only be written once in the test.

Reply:
For now I've removed the uuid, as I agree it's confusing.